src/types/interfaces.ts

```diff
@@ -809,6 +814,18 @@ export interface VideoMediaSession extends IExtendedMediaSession {
 
 export interface ScreenRecordingMediaSession extends IExtendedMediaSession {
   sessionType: SessionTypes.screenRecording;
+  screenMetadatas: ScreenRecordingMetadata[];
+}
+
+export interface ScreenRecordingMetadata {
+  trackId: string;
+  screenId: string;
+  originX: number;
+  originY: number;
+  resolutionX: number;
+  resolutionY: number;
+  primary: boolean;
+  main: boolean;
```

Comment:
Could we document what these properties are?

Reply:
Thanks for reminding me, I need to update some other doc as well.

Reply:
Same as above.